routes/index.routes.js

```diff
@@ -2,17 +2,27 @@ const mongoose = require("mongoose");
 const { Router } = require("express");
 const router = new Router();
 const isLoggedIn = require("../middleware/isLoggedIn");
+const Student = require("../models/model.student");
+const Staff = require("../models/model.staff");
 
 // .get() route ==> signup page
-router.get("/dashboard", isLoggedIn, (req, res) => {
+router.get("/dashboard", isLoggedIn, async (req, res) => {
   const ErrorMessage = req.session.ErrorMessage;
   delete req.session.ErrorMessage;
 
-  
+  const studentsBoys = (await Student.find({ gender: "male" })).length;
```

Comment:
Instead of making 2 times an await call, why dont you use a callback where you filter out all the students in 2 arrays and grab the length?

e.g:
```
const students = await Student.find();
const boys = students.filter(stud => stud.gender === 'male') // array with all the boys
const girls = .... // array with all the girls
```
This reduces the amount of request you're making which makes your application more flexible and faster